python/lsst/daf/butler/_butlerConfig.py

```diff
@@ -93,6 +93,11 @@ def __init__(self, other=None, searchPaths=None):
                     uri = ButlerURI(other, forceDirectory=True)
                 uri.updateFile("butler.yaml")
                 other = uri.geturl()
+            elif uri.scheme == "https":
```

Comment:
I missed this on my ButlerURI cleanup -- no code should be checking URI schemes outside of ButlerURI now. I'll have to clean this up later.